src/main/kotlin/com/zepben/evolve/services/network/tracing/feeder/AssignToFeeders.kt

```diff
@@ -78,7 +78,7 @@ class AssignToFeeders {
         process(terminal.conductingEquipment, ConductingEquipment::addContainer, Feeder::addEquipment, isStopping)
 
     private fun processCurrent(terminal: Terminal, isStopping: Boolean): Unit =
-        process(terminal.conductingEquipment, ConductingEquipment::addCurrentFeeder, Feeder::addCurrentEquipment, isStopping)
+        process(terminal.conductingEquipment, ConductingEquipment::addCurrentContainer, Feeder::addCurrentEquipment, isStopping)
```

Comment:
- Stopping at and excluding equipment with a base voltage value < 1000 should be sufficient unless there are cases where a transformer end's voltage does not match that of the connected equipment on that end. Vice-versa for `AssignToLvFeeders`.
- If a HV feeder trace reaches an LV feeder's equipment which is not that LV feeder's head equipment, is it powering that LV feeder?

Reply:
* Voltages should be being made consistent by the migrators, so just using voltages should be fine
* I would say that if the HV feeder is not connected to the head it is not energizing that LV feeder, even if it is connected to other equipment on the LV feeder. This will be consistent with the HV feeder only being energized by the substation connected to the head. i.e. parallel HV feeders are not energized by both connected substations.